examples/sandbox/index.ts

```diff
@@ -1506,6 +1528,195 @@ $thorchainSignAddLiquidity.on("click", async (e) => {
   }
 });
 
+$thorchainArkeoBondProvider.on("click", async (e) => {
+  e.preventDefault();
+  if (!wallet) {
+    $thorchainNativeResults.val("No wallet?");
+    return;
+  }
+  if (core.supportsThorchain(wallet)) {
+    const res = await wallet.thorchainSignTx({
+      addressNList: core.bip32ToAddressNList(`m/44'/931'/0'/0/0`),
+      chain_id: "thorchain",
```

Comment:
update all paths/chain_id references to arkeo

Reply:
See above